jobs/route_emitter/templates/route_emitter.json.erb

```diff
@@ -4,8 +4,12 @@
   nats_machines = nil
   if_p('diego.route_emitter.nats.machines') do |ips|
     nats_machines = ips.compact
-  end.else_if_link('nats') do
-    nats_machines = link('nats').instances.map { |instance| instance.address }
+  end.else do
```

Comment:
I'm not sure if getting rid of the `if_link('nats')` check would break anything

Reply:
So this is not really an issue.  The previous code if there was no nats link and no nats.machines specified would set nothing for the nats addresses.   And the route_emitter would fail connecting to nats. 

Now this code will if there is no nats.machines specified and there is no nats link this will now fail trying to get the nats link on line 8.   

Both will have the route_emitter failing.  This code is exactly the same as what the route_registrar does and that is why I implemented it as such.  

If you want the route_emitter to fail the same way it does now instead of failing this new way we could change line 7 to match the end.else_if_link('nats') do

Reply:
If I understand Andrew correctly, previously, if there was no nats link and no nats.machines specified, then the code would set nothing for the nats addresses. And then the route_emitter would subsequently fail at runtime because it would be unable to connect to nats as there were no machines/addresses.
This PR will change the behavior so that it will fail at deploy time because it cannot get the required nats link. That is *better* than failing at runtime